daemon/volumes.go

```diff
@@ -138,6 +139,14 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		if label.RelabelNeeded(bind.Mode) {
+			if _, err := os.Stat(bind.Source); err != nil {
```

Comment:
This code was taken from the volume.go.  If MkdirAll, according to the comment there.

```
	// system.MkdirAll() produces an error if m.Source exists and is a file (not a directory),
	// so first check if the path does not exist
```
So I guess if you volume mount a directory over a file, this could cause an issue.